### ISSUE TITLE
GLFW 3.3 Support

### ISSUE THREAD

**Issue by Gnimuc** [JuliaGL/GLFW.jl#181]
I haven't tried it yet, but It looks like Vulkan support on macOS has been improved. Unfortunately, they didn't fix https://github.com/glfw/glfw/issues/1297 in 3.3. Shall we try to revisit the hybrid solution when upgrading to 3.3?

**Comment by musm** [comment]
Do you mind clarifying why BinaryBuilder doesn't work for linux here? 

Does it use the appropriate `host` platform for each out put host ?
I.e. does binarybuilder launch a windows vm to build the windows binaries, and similarily for each other platform?

**Comment by jayschwa** [comment]
I pushed a quick and dirty update to the master branch so that GLFW 3.3 will be used. Hopefully CI passes on all platforms... :crossed_fingers: A new version of GLFW.jl can be tagged once new 3.3 functionality is exposed via Julia functions.

While testing, I encountered a build failure because the upstream CMakeWrapper package used the removed `warn` function. I had to `git clean` and retry `build GLFW`. PR with fix sent: https://github.com/JuliaPackaging/CMakeWrapper.jl/pull/19
